frontend/src/turbo/live-region-stream-action.ts

```diff
@@ -0,0 +1,21 @@
+import { StreamActions, StreamElement } from '@hotwired/turbo';
+import { announce } from '@primer/live-region-element';
+
+export function registerLiveRegionStreamAction() {
+  StreamActions.liveRegion = function liveRegionStreamAction(this:StreamElement) {
+    const message = this.getAttribute('message') || '';
+    const politeness = this.getAttribute('politeness') || 'polite';
+    const delayAttr = this.getAttribute('delay');
+    const delay = delayAttr !== null && !Number.isNaN(Number(delayAttr)) ? Number(delayAttr) : 0;
```

Comment:
You can simplify this as follows:

```suggestion
    const delay = parseInt(this.getAttribute('delay'), 10) || 0;
```

(`parseInt(null) => NaN`, which is `falsey`)

*updated to specify a radix)